PythonRegression/tests/features/steps/blowball_tests_steps.py

```diff
@@ -0,0 +1,89 @@
+from aloe import world, step 
+from util.test_logic import api_test_logic
+from iota import *
+
+import logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+tests = api_test_logic
+
+config = {}
+
+
+@step(r'find transaction is called with the address "([^"]*)" to find milestones')
+def find_transactions_from_coo(step,address):
+    logger.info('Finding milestones')
+    machine = 'machine3'
+    node = 'nodeA'       
+    config['node'] = node
+    
+    api = tests.prepare_api_call(node, machine)
+    config['find_transactions_response'] = api.find_transactions(addresses = [address]) 
```

Comment:
config parameter saves responses?

Reply:
I agree it seems odd, I mean to put a responses variable here to replace config for responses.